backend/src/main/resources/schema.graphqls

```diff
@@ -235,6 +235,7 @@ type Mutation {
     employedInHealthcare: Boolean!
   ): String
   addTestResult(deviceId: String!, result: String!, patientId: String!): String
+  editQueueItem(id: String!, deviceId: String, result: String): TestOrder
```

Comment:
Am I right that this is taking a TestOrder ID? This seems like it's going to make things weird for the client, when all the other queue-related commands refer to the patient ID.

Reply:
At the moment it would be safe to use the patient ID since we only allow one test per patient, but I think test queue id is a better choice. I could imagine doing multiple tests for a single person in the queue. For example two different test types on different devices at once.

Reply:
That would actually be a huge change that would break everything, so being forward compatible for it seems like a premature optimization. :-)